GetBskyStats/run.ps1

```diff
@@ -6,12 +6,17 @@ param($dailyTimer)
 # Write to the Azure Functions log stream.
 Write-Host "PowerShell HTTP trigger function processed a request."
 
-$stats = Invoke-RestMethod -Uri "https://public.api.bsky.app/xrpc/app.bsky.actor.getProfile?actor=jpomfret.co.uk" -Method GET |
-Select-Object -property handle, displayName, avatar, createdAt, description, indexedAt, banner, followersCount, followsCount, postsCount
+$usernames = 'robsewell.com', 'tracisewell.com'

```

Comment:
can you change this back to me - just so when it auto publishes to my function app it still works please :D 

```suggestion
# you can add a list of usernames here too eg:
# $usernames = 'robsewell.com', 'tracisewell.com'
$usernames = 'jesspomfret.co.uk'

```